SofaKernel/modules/SofaMeshCollision/src/SofaMeshCollision/RayTriangleIntersection.cpp

```diff
@@ -21,7 +21,7 @@
 ******************************************************************************/
 #include <SofaMeshCollision/RayTriangleIntersection.h>
 
-#include <SofaMeshCollision/TriangleModel.h>
+#include <SofaMeshCollision/Triangle.h>
```

Comment:
Triangle.h instead of TriangleModel.h?
I thought Triangle.h was the deprecated one?

Reply:
I think those 2 come from a bogus rebase 🤐
Corrected, thanks 👍

Reply:
ok wait for CI and I'll merge that one and move on :)